addons/website_payment/static/src/snippets/s_donation/000.js

```diff
@@ -139,7 +139,14 @@ publicWidget.registry.DonationSnippet = publicWidget.Widget.extend({
                 } else if (amount < parseFloat(minAmount)) {
                     const before = this.currency.position === "before" ? this.currency.symbol : "";
                     const after = this.currency.position === "after" ? this.currency.symbol : "";
-                    errorMessage = _t("The minimum donation amount is %s%s%s", before, minAmount, after);
+                    errorMessage = _t(
+                        "The minimum donation amount is %(currencySymbolWhenBefore)s%(minAmount)s%(currencySymbolWhenAfter)s",
+                        {
+                            currencySymbolWhenBefore: before,
+                            minAmount,
+                            currencySymbolWhenAfter: after,
+                        }
+                    );
```

Comment:
The currency formatting details should not be part of the message IMHO.
```suggestion
                    const minAmountWithCurrency = `$(before)$(minAmount)$(after)`;
                    errorMessage = _t(
                        "The minimum donation amount is %(minAmount)s",
                        {
                            minAmountWithCurrency,
                        }
                    );
```

Reply:
Probably should use `formatCurrency` too, instead of handling the `position` manually here.